src/test/java/io/vertx/test/core/WebsocketTest.java

```diff
@@ -1002,19 +1002,19 @@ public void testFragmentedTextMessage2Hybi13() {
 
   @Test
   public void testFragmentedUnicodeTextMessage2Hybi07() {
-    String messageToSend = TestUtils.randomUnicodeString(65536 + 65536 + 256);
+    String messageToSend = TestUtils.randomUnicodeString(65536 + 256);
```

Comment:
Since these are utf8 strings, they might already be 3 frames long.  I added a new test that uses the maximum message size though.